contributors/devel/cri-validation.md

```diff
@@ -34,15 +41,13 @@ This will
 - Run the tests using `ginkgo`
 - Output the test results to STDOUT
 
-critest connects to `/var/run/dockershim.sock` by default. For other runtimes, the endpoint can be set in two ways:
-
-- By setting flags `--runtime-endpoint` and `--image-endpoint`
-- By setting environment variables `CRI_RUNTIME_ENDPOINT` and `CRI_IMAGE_ENDPOINT`
```

Comment:
Were the environment variables removed, and why?

Reply:
Yep, those environment variables are introduced with lib `github.com/urfave/cli`. But with this lib, critest requires Go installed whenever running validation tests. So we moved to `go test -c` for building critest binary and it doesn't require Go for running critest.